app/forms/hub/create_client_form.rb

```diff
@@ -69,17 +53,12 @@ def initialize(attributes = {})
 
     def save
       return false unless valid?
-      vita_partner_id = attributes_for(:intake)[:vita_partner_id]
-      ActiveRecord::Base.transaction do
-        @intake = Intake.create!(attributes_for(:intake).merge(
-                                   client: Client.create!(vita_partner_id: vita_partner_id),
-                                   preferred_name: calc_preferred_name
-                                 ))
-        @tax_returns_attributes&.each do |_, v|
-          intake.client.tax_returns.create(tax_return_defaults.merge(v)) if create_tax_return_for_year?(v[:year])
-        end
-      end
-      @client = @intake.client
+
+      @client = Client.create(
+        vita_partner_id: attributes_for(:intake)[:vita_partner_id],
+        intake_attributes: attributes_for(:intake),
+        tax_returns_attributes: @tax_returns_attributes.map { |_, v| create_tax_return_for_year?(v[:year]) ? tax_return_defaults.merge(v) : nil }.compact
+      )
```

Comment:
It would be good to include some unit tests to document this behavior (only creates models if _all_ inputs are valid) and to document the basic outputs of the `#save` method (false if not valid, nothing saved, returns truthy client if valid) to document & prevent regression.